tests/components/homekit/test_accessories.py

```diff
@@ -89,14 +89,6 @@ def test_home_accessory(self):
         hass.states.set('homekit.accessory', 'off')
         hass.block_till_done()
 
-        acc = HomeAccessory('hass', 'test_name', 'test_model.demo', 2, {})
-        self.assertEqual(acc.display_name, 'test_name')
-        self.assertEqual(acc.aid, 2)
-        self.assertEqual(len(acc.services), 1)
-        serv = acc.services[0]  # SERV_ACCESSORY_INFO
-        self.assertEqual(
-            serv.get_characteristic(CHAR_MODEL).value, 'Test Model')
```

Comment:
I just realized that we wouldn't/can't cover this case otherwise with the accessory below.
We might have to add it back. Sorry for the confusion.

Reply:
Added back and cleaned up `test_home_accessory`, hopefully you agree with my changes.